src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -288,7 +294,6 @@ private Case buildCaseFromSourceCaseAndEvent(CollectionCase newCollectionCase, C
     newCase.setOa(sourceCase.getOa());
     newCase.setPrintBatch(sourceCase.getPrintBatch());
     newCase.setSurvey(sourceCase.getSurvey());
-    newCase.setMetadata(metadataFromSourceCase(sourceCase.getMetadata()));
```

Comment:
by removing this and using the `getMetadataFromEventOrSourceCase` we are potentially going to lose the `channel` metadata which would have been copied over - I see that as a bug

Reply:
actually, I checked the original code, and it had the 'bug' too